shell/common/rasterizer.cc

```diff
@@ -170,7 +171,11 @@ void Rasterizer::Draw(fml::RefPtr<Pipeline<flutter::LayerTree>> pipeline) {
   RasterStatus raster_status = RasterStatus::kFailed;
   Pipeline<flutter::LayerTree>::Consumer consumer =
       [&](std::unique_ptr<LayerTree> layer_tree) {
-        raster_status = DoDraw(std::move(layer_tree));
+        if (discardCallback && discardCallback(*layer_tree.get())) {
+          raster_status = RasterStatus::kSuccess;
```

Comment:
It might make sense to update the docs on the `RasterStatus::kSuccess` enum to reflect that a layer tree could be discarded as well, or maybe even create a new raster status `kDiscarded`.

Reply:
That raster_status value never leaves `Rasterizer::Draw`. But I understand the point.